static/app/views/alerts/rules/metric/ruleForm.spec.tsx

```diff
@@ -234,7 +234,7 @@ describe('Incident Rules Form', () => {
       });
 
       await waitFor(() =>
-        expect(screen.getByTestId('alert-total-events')).toHaveTextContent('Total5')
+        expect(screen.findByTestId('alert-total-events')).toHaveTextContent('Total5')
```

Comment:
need an await here and remove the waitFor since its no longer necessary

Reply:
ofc, thx! it is clearly too late in my time zone 🙈